meta-resin-common/recipes-core/images/resin-image.bb

```diff
@@ -28,7 +28,6 @@ IMAGE_FEATURES_append = " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'development-image', 'debug-tweaks', '', d)} \
     splash \
     ssh-server-dropbear \
-    read-only-rootfs \
```

Comment:
Haven't had a chance to test whether this is still required, adding back for now.